### DISCUSSION TITLE
Indentation of context code in diff view is totally broken

### DISCUSSION THREAD

**Discussion by dead10ck** [community/community#47204] [Mobile] [answered]
### Select Topic Area

Bug

### Body

In the diff view, code that surrounds the changed text is completely unindented, making it very difficult to read.

![Screenshot_20230106-101118_GitHub](https://user-images.githubusercontent.com/846275/218374681-5d099e35-f30e-4754-9c98-d3fbb88c9cdc.png)
![Screenshot_20230106-101056_GitHub](https://user-images.githubusercontent.com/846275/218374683-7fb79d21-a739-4ab0-813f-e1ba4bfecbc6.png)

**Comment by mattruggio** [comment, marked as answer]
Hey @dead10ck thanks for the heads up on this issue.  I can get an internal bug filed for this for further inspection.  Could you please provide your device manufacturer, model, and OS version?

**Reply by dead10ck** [reply]
Thanks @mattruggio. It's a OnePlus 9 Pro, running LineageOS 20.0, but I also saw it in OxygenOS, whatever the latest version was a few months ago.

**Reply by jsoref** [reply]
Google Pixel 3A / Android 12

![Screenshot_20230220-120115.png](https://user-images.githubusercontent.com/2119212/220165760-c9024c02-75f4-47f8-a27c-ac1bc15425d8.png)

https://github.com/cert-manager/cert-manager/pull/5814

**Reply by dead10ck** [reply]
Is there any update on this?

**Reply by dead10ck** [reply]
Pinging @mattruggio ?

**Reply by mattruggio** [reply]
I just checked in on this: it has been internally triaged and is in the todo queue.  Unfortunately I am not sure on the priority of this but I will re-bump it up for visibility!

**Reply by dead10ck** [reply]
Thanks a lot @mattruggio I appreciate it! 🙏

**Reply by dead10ck** [reply]
@mattruggio it looks like as of 1.115.0, this issue has been fixed! Thanks so much for following this up. ❤️

**Comment by malikAdnanm220** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Body
> 
> In the diff view, code that surrounds the changed text is completely unindented, making it very difficult to read.
> 
> ![Screenshot_20230106-101118_GitHub](https://user-images.githubusercontent.com/846275/218374681-5d099e35-f30e-4754-9c98-d3fbb88c9cdc.png)
> ![Screenshot_20230106-101056_GitHub](https://user-images.githubusercontent.com/846275/218374683-7fb79d21-a739-4ab0-813f-e1ba4bfecbc6.png)
>
